quantum/process_keycode/process_unicode_common.c

```diff
@@ -19,42 +19,46 @@
 #include <string.h>
 #include <ctype.h>
 
-static uint8_t input_mode;
-uint8_t mods;
+unicode_config_t unicode_config;
+static uint8_t saved_mods;
+#ifdef AUDIO_ENABLE
+  #ifdef UNICODE_LINUX_SONG
+    float linux_song[][2] = UNICODE_LINUX_SONG;
```

Comment:
```suggestion
    float song_lnx[][2] = UNICODE_SONG_LNX;
```
Same goes for the arrays.